selectors.yml

```diff
@@ -9,11 +9,13 @@ selectors:
   80001: 12532609583862916517 # Polygon Mumbai
   421613: 6101244977088475029  # Arbitrum Goerli
   11155111: 16015286601757825753 # Sepolia
+  1112: 9284632837123596123 #wemix
```

Comment:
Please order by chain ID (and add a space after #). And this is a testnet, so please call it wemix testnet